pyam/_aggregate.py

```diff
@@ -51,6 +53,39 @@ def _aggregate(df, variable, components=None, method=np.sum):
     return _group_and_agg(_df, [], method)
 
 
+def _aggregate_recursive(df, variable, components=None, method=np.sum):
+    _dfout = None
```

Comment:
the variable `_dfout` is not very pythonic...

Reply:
renamed _dfout to _df_aggregated, hopefully this is more pythonic